packages/swapper/src/swappers/RelaySwapper/utils/getLatestRelayStatusMessage.ts

```diff
@@ -0,0 +1,23 @@
+import { RelayStatusMessage } from '../constant'
+import type { RelayStatus } from './types'
+
+export const getLatestRelayStatusMessage = (status: RelayStatus): string => {
+  const { status: statusValue, details } = status
+
+  switch (true) {
+    case statusValue === 'waiting':
+      return RelayStatusMessage.WaitingForDeposit
+    case statusValue === 'pending' && details?.includes('Could not fill request'):
+    case statusValue === 'delayed':
+      return RelayStatusMessage.Retrying
+    case statusValue === 'pending' &&
+      (details?.includes('Generating solution') || details?.includes('Filling solution')):
```

Comment:
q: do we need the right side here? Regardless of details, sounds like any pending is a deposit detected?

Reply:
Happy with it as I couldn't find any different message but to clarify: any `pending` are not a deposit detected, but any `pending` that are not `Could not fill request` probably are!

Removed the right side in [180cbd0](https://github.com/shapeshift/web/pull/9389/commits/180cbd0e95d02a4926da953828c0e1ff46c7f695)